src/respond.rs

```diff
@@ -75,7 +75,7 @@ use crate::{Request, ResponseTemplate};
 /// `Respond` that propagates back a request header in the response:
 ///
 /// ```rust
-/// use http_types::headers::HeaderName;
+/// use hyper::header::HeaderName;
```

Comment:
Can't we import from `http` here? I don't think we need to import from `hyper`.

Reply:
I think it would require an extra (dev) dependency, but probably worth it.

Reply:
It would, but `http` is already in our (non-dev) dependency tree, so it doesn't make an actual different.

Reply:
Yeah, just one more version to keep in sync with the test in the manifest.